composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "lmc/matej",
+    "description": "API Client for Matej recommendation engine",
+    "type": "library",
+    "license": "MIT",
+    "minimum-stability": "stable",
+    "authors": [
+        {
+            "name": "LMC s.r.o.",
+            "homepage": "https://github.com/lmc-eu"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Lmc\\Matej\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Lmc\\Matej\\": "tests/"
+        }
+    },
+    "require": {
```

Comment:
co PHP verze?

Reply:
Tohle poresim, az budeme resit tu transpilaci pro 5.6 - ted jeste nevim, jak to bude.